packages/analytics/RNFBAnalytics.podspec

```
@@ -41,6 +41,7 @@ Pod::Spec.new do |s|
   # Firebase dependencies
   if defined?($RNFirebaseAnalyticsWithoutAdIdSupport) && ($RNFirebaseAnalyticsWithoutAdIdSupport == true)
     Pod::UI.puts "#{s.name}: Not installing FirebaseAnalytics/IdentitySupport Pod, no IDFA will be collected."
+    s.dependency           'FirebaseAnalytics/Core', firebase_sdk_version
```

In the new pod arrangement for FirebaseAnalytics pods + sub-pods, combined with our desire to allow folks to have fine-grained control with Podfile variable toggles so that all parts are optional, we need to start with `FirebaseAnalytics/Core` for *all* arrangements and then go from there

I think it may work to have the `Core` sub-spec in the conditional because of transitive dependencies bringing it in some of the conditional branches where it is needed, but...

I think it's most correct to always start with it as a dependency, outside of all conditionals, and then go

What do you think @radko93 ? I think this is most correct and if I don't hear back I'm going to commit and release it so we don't have the broken 22.3.0 version out too long...


```suggestion
  s.dependency           'FirebaseAnalytics/Core', firebase_sdk_version
  if defined?($RNFirebaseAnalyticsWithoutAdIdSupport) && ($RNFirebaseAnalyticsWithoutAdIdSupport == true)
    Pod::UI.puts "#{s.name}: Not installing FirebaseAnalytics/IdentitySupport Pod, no IDFA will be collected."
```